Added search command

ripSquid opened the pull request:
I added a search command to be able to search for a song and play it automatically inside discord.

I am unsure if powercord spotify permissions are needed. So just add a change if needed.

ripSquid committed:
Added search command

cyyynthia commented on src/Powercord/plugins/pc-spotify/commands/search.js:
```diff
@@ -0,0 +1,38 @@
+const SpotifyAPI = require('../SpotifyAPI');
+
+module.exports = {
+    command: 'search',
```
This command is already in use by [another plugin](https://github.com/Juby210/quick-search), which would cause a conflict

ripSquid replied:
Renamed to find

cyyynthia commented on src/Powercord/plugins/pc-spotify/commands/search.js:
```diff
@@ -0,0 +1,38 @@
+                    'uris': [trackURL]
+                })
+            } else {
+                return {
```
This return will never be seen by our command handler since it is inside a promise

ripSquid replied:
Also returned the promise
`return SpotifyAPI.search(args.join(' '), 'track', 1).then((body) => {`

cyyynthia reviewed (changes requested):
There's a few things that need attention, and there needs to be some changes regarding linting as you can see from the [Travis build logs](https://travis-ci.org/github/powercord-org/powercord/builds/710493973#L227)

ripSquid commented:
I think I have fixed the linting issues and also one forgotten import. I think I have also fixed the return. Someone suggested the command "find" so if that is available I'm all good.

ripSquid committed:
Renamed it to find and fixed some other changes

ripSquid committed:
Fixed returning stuff from SpotifyAPI.play
